src/site/components/events_list.drupal.liquid

```diff
@@ -6,46 +6,49 @@
   {% endif %}
   {% assign sortedEventsArray = eventsArray | eventSorter %}
   {% for event in sortedEventsArray %}
-  {% if event.title.length %}
-  {% assign count = count | plus:1 %}
-  <div class="vads-u-margin-bottom--2">
-    <h4><a href="{{event.entityUrl.path}}">{{event.title}}</a></h4>
-    <p>{{event.fieldDescription}}</p>
-    <div class="usa-grid usa-grid-full">
-      {% if event.fieldDate.value %}
-      <div class="usa-width-one-sixth">
-        <strong>When:</strong>
+    {% if event.title.length %}
+      {% assign count = count | plus:1 %}
+      <div class="vads-u-margin-bottom--2">
+        <h4>
+          <a href="{{event.entityUrl.path}}">{{event.title}}</a>
+        </h4>
+        <p>{{event.fieldDescription}}</p>
+        <div class="usa-grid usa-grid-full">
+          {% if event.fieldDate.value %}
+            <div class="usa-width-one-sixth">
+              <strong>When:</strong>
+            </div>
+            <div class="usa-width-five-sixths">
+              <span>{{ event.fieldDate.value | humanizeDate }}</span><br>
+              <span>{{ event.fieldDate.value | humanizeTime }}
+                –
+                {{ event.fieldDate.endValue | humanizeTime }}</span>
+            </div>
+          {% endif %}
+        </div>
+        <div class="usa-grid usa-grid-full">
+          {% if event.fieldFacilityLocation.entity.entityUrl.path %}
+            <div class="usa-width-one-sixth">
+              <strong>Where:</strong>
+            </div>
+            <div class="usa-width-five-sixths">
+              <p>
+                <a href="{{event.fieldFacilityLocation.entity.entityUrl.path}}">{{event.fieldFacilityLocation.entity.title}}</a>
```

Comment:
@ethanteague Is there a guarantee of always having a location?

Reply:
Good q, and I see what you mean - I moved that conditional check up to grab the entire wrapper, @swirtSJW